.github/CODEOWNERS

```diff
@@ -64,9 +64,9 @@ CONTRIBUTING.md                                                   @swissspidy
 #assets/src/edit-story/elements/image/                             @cvolzke4 @diegovar @jhtin @joannag6 @winterDroid
 #assets/src/edit-story/elements/video/                             @cvolzke4 @diegovar @jhtin @joannag6 @winterDroid
 
-## Movable
-#assets/src/edit-story/components/movable/                         @miina
-#assets/src/edit-story/elements/image/editCropMovable.js           @miina
-#assets/src/edit-story/elements/shared/editPanMovable.js           @miina
-#assets/src/edit-story/components/canvas/singleSelectionMovable.js @miina
-#assets/src/edit-story/components/canvas/multiSelectionMovable.js  @miina
+## Moveable
```

Comment:
@swissspidy -- FI: changed these based on renaming all `movable` to `moveable` for consistency. These paths are actually a bit outdated anyway (e.g. there's only the image and not video/media folder), however, didn't change this since not really related to this PR.